app/customers.py

```diff
@@ -0,0 +1,28 @@
+import math
+
+
+class Customer:
+    def __init__(self, user_info, fuel_price):
+        self.name = user_info["name"]
+        self.product_cart = user_info["product_cart"]
+        self.location = user_info["location"]
+        self.money = user_info["money"]
+        self.car = user_info["car"]
+        self.fuel_price = fuel_price
```

Comment:
I don't think fuel price is correct attribute for Customer. You know, fuel price not connect to Customer. If fuel price changed you need to change all your Customer